core/base/inc/TString.h

```diff
@@ -138,6 +138,12 @@ friend Bool_t operator==(const TSubString &s1, const char *s2);
 //////////////////////////////////////////////////////////////////////////
 class TString {
 
+public:
+
+using size_type = Ssiz_t;
```

Comment:
Can you move this in the existing public section (next to `enum EStripType`)

Reply:
Sure!